tests/installation/partitioning_raid.pm

```diff
@@ -31,6 +31,11 @@ sub addpart {
     else                         { die 'Unknown argument'; }
 
     assert_screen "expert-partitioner";
+    unless (is_sle('<15') || is_leap('<15.1') || is_tumbleweed) {
```

Comment:
Could we at least put this monster condition in one place, so when it reaches TW, we don't have to change it in 3 places or potentially more?
Other than that LGTM.